### PR TITLE
explain how to modify list of browsers in the plugins file

### PR DESCRIPTION
**Pull request by bahmutov**
related to new feature https://github.com/cypress-io/cypress/pull/5068

### CODE REVIEW

**Comment by jennifer-shehane** [comment]
This is a great writeup - I have some more places I want to mention this, but I moved this initial full guide into the 'Launching Browsers' guide, where we talk about the available browsers. I think this will be closer to the intent - if someone were looking to change what browsers are launched. 

Gonna see if I can loop this in to other places in the docs, like the plugins - it's a bit awkward cause it doesn't use the plugin events. 🤔

**Comment by jennifer-shehane** [comment]
Found it! I knew we have a specific guide showing how to manipulate the config in plugins. https://docs.cypress.io/api/plugins/configuration-api.html

**Comment by bahmutov** [comment]
Super

**Comment by jennifer-shehane** [comment]
I want to update the image of the resolved configuration whenever the PR is definitely merged in 🙃 https://docs.cypress.io/guides/references/configuration.html#Resolved-Configuration

File: source/guides/guides/launching-browsers.md [hunk 1]
```diff
@@ -81,9 +81,89 @@ Cypress will automatically detect the type of browser supplied and launch it for
...
+Once selected, the Brave browser is detected using the same approach as any other browser of the `chrome` family.
+
+{% imgTag /img/guides/plugins/brave-running-tests.png "Brave browser executing end-to-end tests" %}
+
+If you modify the list of browsers, you can see the {% url "resolved configuration" configuration#Resolved-Configuration %} in the **Settings** tab of the Test Runner.
+
 ## Unsupported Browsers
 
-Many browsers such as Firefox, Safari, and Internet Explorer are not currently supported. Support for more browsers is on our roadmap. You can read an exhaustive explanation about our future cross browser testing strategy {% issue 310 'here' %}.
+Many browsers such as Firefox, Safari, and Internet Explorer are not currently supported. While it may be possible to modify the list of browsers found and run Cypress within them - we do not gaurantee full functionality. Support for more browsers is on our roadmap. You can read an exhaustive explanation about our future cross browser testing strategy {% issue 310 'here' %}.
```
Comment (bahmutov) [inline source/guides/guides/launching-browsers.md]: `gaurantee` is an exotic fruit

Comment (jennifer-shehane) [reply source/guides/guides/launching-browsers.md]: That's a hard one, gonna add that to the linter 😅

**Pull request merged**